### PR TITLE
contracts-bedrock: StandardBridge pausability

### PR DESCRIPTION
**Pull request by tynes**
**Description**

Enable extended pause functionality in the standard bridge.
Includes updates to tests and the deploy script.

### CODE REVIEW

File: packages/contracts-bedrock/src/L1/L1StandardBridge.sol
Comment (tynes) [reply packages/contracts-bedrock/src/L1/L1StandardBridge.sol:88]: This is a good idea but we don't do it throughout the codebase. It is better to be standard

File: packages/contracts-bedrock/src/L1/L1CrossDomainMessenger.sol
Comment (tynes) [reply packages/contracts-bedrock/src/L1/L1CrossDomainMessenger.sol]: We know this is safe based on the `upgradeAndCall` pattern

File: packages/contracts-bedrock/test/L1/L1StandardBridge.t.sol
Comment (maurelian) [inline packages/contracts-bedrock/test/L1/L1StandardBridge.t.sol:42]: We need a test or tests to show that none of the withdrawal methods are callable when the bridge is paused.

Comment (tynes) [reply packages/contracts-bedrock/test/L1/L1StandardBridge.t.sol:42]: Good callout

File: packages/contracts-bedrock/test/L1/L1StandardBridge.t.sol
Comment (maurelian) [reply packages/contracts-bedrock/test/L1/L1StandardBridge.t.sol]: Thanks buddy, appreciate it.

Comment (tynes) [reply packages/contracts-bedrock/test/L1/L1StandardBridge.t.sol]: Need to update this to pause withdrawals and not deposits

Comment (tynes) [reply packages/contracts-bedrock/test/L1/L1StandardBridge.t.sol]: This is close but not exactly how to perform withdrawals

Comment (tynes) [reply packages/contracts-bedrock/test/L1/L1StandardBridge.t.sol]: You were more correct than I expected. The only thing you did wrong was you used the legacy methods. We need to test both the legacy methods and the modern methods. The modern methods are implemented on the `StandardBridge` and are symmetric across L1 and L2. They are `finalizeBridgeETH` and `finalizeBridgeERC20`

Comment (tynes) [reply packages/contracts-bedrock/test/L1/L1StandardBridge.t.sol]: Your tests were very useful and helped me. They were an inspiration for my own.

File: packages/contracts-bedrock/semver-lock.json
Comment (tynes) [inline packages/contracts-bedrock/semver-lock.json]: Hmm why is the L1 xdm changing here?

File: packages/contracts-bedrock/src/L2/L2StandardBridge.sol
Comment (maurelian) [inline packages/contracts-bedrock/src/L2/L2StandardBridge.sol]:
```suggestion
    string public constant version = "1.6.0";
```

File: packages/contracts-bedrock/src/L1/L1StandardBridge.sol
Comment (tynes) [reply packages/contracts-bedrock/src/L1/L1StandardBridge.sol:74]: Are you sure about this?

File: packages/contracts-bedrock/test/L1/L1StandardBridge.t.sol
Comment (maurelian) [inline packages/contracts-bedrock/test/L1/L1StandardBridge.t.sol:70]: This test file pretty consistently adds `@dev` comments on the test functions, I think it's worth doing on these too.

Comment (maurelian) [reply packages/contracts-bedrock/test/L1/L1StandardBridge.t.sol:70]: @coderabbitai perhaps you could go through the diff in this file and suggest a natspec comment for each new test function? It should be in the format `/// @dev` and should provide a brief description of what is being tested.

**Review by maurelian — APPROVED** [review]
Approving with one comment/suggestion that IMO should be addressed before merging.

File: packages/contracts-bedrock/test/L1/L1StandardBridge.t.sol
Comment (maurelian) [reply packages/contracts-bedrock/test/L1/L1StandardBridge.t.sol:70]: Any chance you could format that as a patch file to make things easier for us?

Comment (maurelian) [reply packages/contracts-bedrock/test/L1/L1StandardBridge.t.sol:70]: no, the patch shouldn't delete any code.
Provide a patch that only adds the dev comments please.

File: packages/contracts-bedrock/test/L1/L1StandardBridge.t.sol
Comment (tynes) [reply packages/contracts-bedrock/test/L1/L1StandardBridge.t.sol]: I did add the `@dev` comments in a future commit

File: packages/contracts-bedrock/test/L1/L1StandardBridge.t.sol
Comment (tynes) [reply packages/contracts-bedrock/test/L1/L1StandardBridge.t.sol:70]: Thank you for the help. I have added the dev comments. In the future, it would be most useful to receive a commit suggestion as a comment so that I can add the patch directly through the ui

**Pull request merged**
